server/schedule/replica_checker.go

```diff
@@ -224,7 +226,7 @@ func (r *ReplicaChecker) checkBestReplacement(region *core.RegionInfo) *Operator
 	}
 	// Make sure the new peer is better than the old peer.
 	if newScore <= oldScore {
-		log.Debugf("[region %d] newScore %f is not better than oldScore %f", region.GetID(), newScore, oldScore)
+		log.L().Debug("no better peer", zap.Uint64("region", region.GetID()), zap.Float64("new score", newScore), zap.Float64("old score", oldScore))
```

Comment:
Inconsistent naming style (space separated instead of kebab)

Reply:
BTW, for some common use like `regionID` and `storeID`, I think we need to make them consistent.
/cc @nolouch